lib/pages/contacts_tab/contacts_tab_body_view.dart

```diff
@@ -25,6 +26,16 @@ class ContactsTabBodyView extends StatelessWidget {
             color: Colors.black.withOpacity(0.15),
           ),
         ),
+        SliverToBoxAdapter(
+          child: ContactsWarningBannerView(
```

Comment:
duplicate code, ContactsWarningBannerView should have own controller and contacts manager